src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/HashHelper.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.companieshouse.overseasentitiesapi.utils;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Base64;
+
+public class HashHelper {
+
+    private String salt;
+
+    public HashHelper(String salt) {
+        this.salt = salt;
+    }
+
+    public String encode(String plain) throws NoSuchAlgorithmException {
+        return base64Encode(sha1Digest(plain)).replace("=", "");
+    }
+
+    private String base64Encode(final byte[] bytes) {
+        return Base64.getUrlEncoder().encodeToString(bytes);
+    }
+
+    private byte[] sha1Digest(final String plain) throws NoSuchAlgorithmException {
```

Comment:
SHA-2 and SHA-3 are more secure. Any reason why those algorithms aren't being used?

Reply:
I'd agree, unless there is a need to use SHA1 due to wider constraints, why not use SHA-2 or better 3?

Reply:
👍 thanks for clarifying.